cypress/support/fragments/data-export/dataExportResults.js

```diff
@@ -134,4 +134,43 @@ export default {
       }),
     );
   },
+
+  verifyLastLog(fileName, status) {
+    const result = {
+      fileName: MultiColumnListCell({ row: 0, columnIndex: 0 }),
+      status: MultiColumnListCell({ row: 0, columnIndex: 1 }),
+    };
+
+    cy.do([result.status.is({ content: status })]);
+
+    cy.do(
+      result.fileName.perform((element) => {
+        const regex = new RegExp(`${fileName.slice(0, -4)}-\\d+.mrc`);
+        expect(element.innerText).to.match(regex);
+      }),
+    );
+  },
+
+  verifyFileNameIsDisabled(rowNum) {
+    const cellLocator = `[data-row-inner="${rowNum}"]>div>span`;
```

Comment:
It's preferable to use interactors instead of cypress native locators

Reply:
Fixed where possible.